### PR TITLE
Allow delta_clifford=1 in all single qubit RB

### PR DESCRIPTION
**Pull request by TheoLaudatQM**

### CODE REVIEW

**Review by deanpoulos — APPROVED** [review]

**Comment by deanpoulos** [comment]
Hi @TheoLaudatQM, the logic of the program with `depth_target` and so on is a little hard to follow. However, I was able to validate both the old and new RB using `qua-qsim` and couldn't find any huge difference between the output RB curves of the programs

**Comment by TheoLaudatQM** [comment]
Great, I am merging then!
